backend/aws_helpers/dynamo_db_utils/constants.py

```diff
@@ -0,0 +1,13 @@
+EXECUTION_TABLE_NAME = "execution-table"
+USERPROGRESS_TABLE_NAME = "userprogress_table"
+FILE_UPLOAD_TABLE_NAME = "dlp-file-upload-table"
```

Comment:
Hey @farisdurrani, daniel and I were talking and we might not really need the `FILE_UPLOAD_TABLE_NAME` dynamodb table. can you go ahead and remove that dynamodb resource in AWS and also in our terraform config

Reply:
I can do that

Reply:
Awesome. Can you update your PR with that fix? @farisdurrani